packages/app/src/components/swap/exit.tsx

```diff
@@ -64,19 +65,24 @@ const ExitPool: FC<Props> = ({ name, symbol, lpTokenBalance }) => {
     }
   );
 
-  const { state: exitPoolTx, send: removeLiquidity } = useContractFunction(contracts[name].lpToken, "removeLiquidity", {
+  const { state: exitPoolTx, send: removeLiquidity } = useContractFunction(contracts[name].tenderSwap, "removeLiquidity", {
     transactionName: `exit t${symbol}/${symbol} Liquidity Pool`,
   });
 
   const singleTokenOutAddress = selectedToken === symbol ? addresses[name].token : addresses[name].tenderToken;
 
-  const [singleOut] = useCalculateRemoveLiquidityOneToken(
-    addresses[name].lpToken,
+  const singleOut = useCalculateRemoveLiquidityOneToken(
+    addresses[name].tenderSwap,
     utils.parseEther(lpSharesInput || "0"),
     singleTokenOutAddress
   );
 
-  const [tenderOut, tokenOut] = useCalculateRemoveLiquidity(addresses[name].lpToken, utils.parseEther(lpSharesInput || "0"));
+  const [tenderOut, tokenOut] = useCalculateRemoveLiquidity(addresses[name].tenderSwap, utils.parseEther(lpSharesInput || "0"));
+
+  useEffect(() => {
+    setTokenOutput(utils.formatEther(tokenOut || "0"));
+    setTenderOutput(utils.formatEther(tenderOut || "0"))
+  }, [tenderOut, tokenOut]);
```

Comment:
@kyriediculous this shouldn't be needed. When `tenderOut` or `tokenOut` changes, the component rerenders, and every value should update which uses it, e.g. before this change `value={utils.formatEther(tokenOut) || "0"}` in the jsx.

Reply:
gotcha , thanks !